SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -600,6 +603,35 @@ void USpatialNetDriver::QueryGSMToLoadMap()
 	GlobalStateManager->QueryGSM(QueryDelegate);
 }
 
+void USpatialNetDriver::OnActorSpawned(AActor* Actor)
+{
+	if (!Actor->GetIsReplicated())
+	{
+		return;
+	}
+
+	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton | SPATIALCLASS_NotSpatialType))
+	{
+		return;
+	}
+
+	if (Actor->GetLocalRole() != ROLE_Authority)
+	{
+		return;
+	}
+
+	if (USpatialStatics::IsActorGroupOwnerForActor(Actor))
+	{
+		return;
+	}
```

Comment:
Is this a pattern we're using in the GDK? What's justification vs. just using once conditional with ORs?

Reply:
I am happy to change it to ORs, but I do think we have this pattern elsewhere as well.